DependencyInjection/Configuration.php

```diff
@@ -26,8 +26,12 @@ class Configuration implements ConfigurationInterface
      */
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder();
-        $rootNode = $treeBuilder->root('dunglas_angular_csrf');
+        $treeBuilder = new TreeBuilder('dunglas_angular_csrf');
```

Comment:
The argument should be set only when `getRootNode` exists (please move the instantiation into each code branch).

Reply:
Interesting! I saw this exact solution in another package. Didn't consider this would be wrong in SF < 4.3. Time to have a look at the other packages as well then, thanks!

Reply:
Updated the PR! :)

Is this the solution you're looking for?

Reply:
Yes perfect